src/main/java/bio/terra/service/profile/flight/delete/ProfileDeleteFlight.java

```diff
@@ -33,6 +41,9 @@ public ProfileDeleteFlight(FlightMap inputParameters, Object applicationContext)
         CloudPlatformWrapper.of(
             inputParameters.get(JobMapKeys.CLOUD_PLATFORM.getKeyName(), String.class));
 
+    AzureStorageMonitoringStepProvider azureStorageMonitoringStepProvider =
+        new AzureStorageMonitoringStepProvider(monitoringService);
+
```

Comment:
If this is only used inside the `if` below, then it could be defined there instead. That limits its scope and prevents unneeded code from running.